src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -148,7 +148,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 ' PERF: if allocations here become nuisance, consider caching the TupleTypeSymbol
                 '       in the type symbols that can actually be tuple compatible
-                srcElementFields = TupleTypeSymbol.Create(DirectCast(srcType, NamedTypeSymbol)).TupleElementFields
+                srcElementFields = TupleTypeSymbol.Create(DirectCast(srcType, NamedTypeSymbol)).TupleDefaultElementFields
             End If
```

Comment:
Consider using `If` to get `TupleTypeSymbol` and access `TupleDefaultElementFields` outside of `If`.